internal/install/command.go

```diff
@@ -41,17 +42,26 @@ var Command = &cobra.Command{
 		logLevel := configAPI.GetLogLevel()
 		config.InitFileLogger(logLevel)
 
-		if err := checkNetwork(client.NRClient); err != nil {
+		sg := segment.New()
+
+		err := assertProfileIsValid(config.DefaultMaxTimeoutSeconds, sg)
```

Comment:
@NRhzhao it looks like we may get an error below, could we start with an event to indicate we're starting the cli init?
Also could we get the accountID and region from the env var? That would be valuable information in case we get any of the errors afterward.

Reply:
All the tracking is done inside the assertProfileIsValid function since we want to have specific event type fo different errors that occurred.